### PR TITLE
Update test dependencies

### LINKED ISSUE #62: chore: Update dependencies
Python is getting updated, [related](https://github.com/kytos-ng/kytos/issues/448). Some dependencies from this NApp need to be updated.

### PR DESCRIPTION
**Pull request by Alopalao**
Closes #62

### Summary

Updated test dependencies
Depends in kytos [PR](https://github.com/kytos-ng/kytos/pull/460)

### Local Tests

Test by running with updated kytos

### End-to-End Tests
TBA

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-image: python:3.9
+image: python:3.11
```
Comment (viniarck) [inline .gitlab-ci.yml]: You can nuke this `.gitlab-ci.yml` as far as I know it's not being used

**Review by viniarck — APPROVED** [review]
LGTM. Leaving it pre-approved. 

Let's try to land all PRs in the same day, once we have the others approved too it'll get merged.

This is also assuming that pinned branch will be reverted and scrutinizer.yml that we spoke, thanks, Aldo.

File: .gitlab-ci.yml (hunk 1 above)
Comment (Alopalao) [reply .gitlab-ci.yml]: Nuked

**Review by viniarck — APPROVED** [review]
LGTM. Leaving it pre-approved to land soon

**Pull request merged**